.circleci/config.yml

```diff
@@ -0,0 +1,73 @@
+---
+# CI Workflow for the Forward Justice/Traffic Stops application
+
+version: 2.1
+
+commands:
+  restore_cache_cmd:
+    steps:
+      - restore_cache:
+          keys:
+              - v7-pip-dependencies-{{ checksum "requirements/test/test.txt" }}-{{ checksum "requirements/base/base.txt" }}
+              # fallback to using the latest cache if no exact match is found
+              - v7-pip-dependencies-
+  save_cache_cmd:
+    steps:
+      - save_cache:
+          paths:
+            - "env"
+          key: v7-pip-dependencies-{{ checksum "requirements/test/test.txt" }}-{{ checksum "requirements/base/base.txt" }}
+orbs:
+  slack: circleci/slack@3.4.2
+
+jobs:
+  test:
+    machine:
+      image: ubuntu-1604:202007-01
```

Comment:
Interesting- why this image?

Reply:
No particular reason. It was one of the available machine images that circle provides. These images come with docker and docker-compose so it is convenient.